nodes/move-object/move-object.html

```diff
@@ -0,0 +1,78 @@
+<!-- BEGIN PUT OBJECT -->
+<script type="text/x-red" data-template-name="Move Object">
+    <div class="form-row">
+        <label for="node-input-name"><i class="fa fa-tag"></i> <span data-i18n="Name">&nbsp;</span>Name: </label>
+        <input type="text" id="node-input-name" data-i18n="[placeholder]Name">
+    </div>
+    <div class="form-row">
+        <label for="node-input-conf"><code><i class="fa fa-cogs"></i></code> <span data-i18n="Configuration">&nbsp;</span>Client: </label>
+        <input type="text" id="node-input-conf">
+    </div>
+    <div class="form-row">
+        <label for="node-input-bucket"><code><i class="fa fa-database"></i></code> <span data-i18n="Bucket">&nbsp;</span>Bucket: </label>
+        <input type="text" id="node-input-bucket" data-i18n="[placeholder]Bucket">
+    </div>
+    <div class="form-row">
+        <label for="node-input-key"><code><i class="fa fa-magic"></i></code> <span data-i18n="Key">&nbsp;</span>Key: </label>
+        <input type="text" id="node-input-key" data-i18n="[placeholder]Object Key">
+    </div>
+     <div class="form-row">
+        <label for="node-input-sourcebucket"><code><i class="fa fa-magic"></i></code> <span data-i18n="Source Bucket">&nbsp;</span>Source Bucket: </label>
+        <input type="text" id="node-input-sourcebucket" data-i18n="[placeholder]Source Bucket">
+    </div>
+    <div class="form-row">
+        <label for="node-input-sourceobject"><code><i class="fa fa-magic"></i></code> <span data-i18n="Source Object">&nbsp;</span>Source Object: </label>
```

Comment:
The icons for the inputs should represent the input as close as possible. Supported icons by Node-RED can be found [here](https://fontawesome.com/v4/icons/).